acceptance-tests/script/wait.sh

```diff
@@ -220,7 +220,18 @@ EOF
 echo " ... HLR preactivated rofiles loaded into prime"
 
 
+##
+## Creating pub-sub subscriptions for recurring payments tests (plans).
+## Have to create one subscription per test, as the tests runs in parallel.
+##
+
+echo "Creating pub-sub subscriptions for recurring payment tests"
+curl -X PUT -H "Content-Type: application/json" -d '{"topic":"projects/'${GCP_PROJECT_ID}'/topics/stripe-event","ackDeadlineSeconds":10}' pubsub-emulator:8085/v1/projects/${GCP_PROJECT_ID}/subscriptions/stripe-event-okhttp-purchase-ok-sub
```

Comment:
This is so much repeated code that it now arguably would make sense to factor this out into a function.   Or not. Your call.

Reply:
Will try to move the `curl` call into a function.